ntbs-service/Pages/Notifications/OverviewPartials/_TreatmentEventsOverviewPartial.cshtml

```diff
@@ -48,14 +48,20 @@
             var treatmentEvents = Model.GroupedTreatmentEvents[episodeNumber];
             
             <div>
-                <h4 class="episode-grouping-overview-heading">Episode @episodeNumber</h4><span>@treatmentEvents.First().FormattedEventDate - @treatmentEvents.Last().FormattedEventDate </span>
+                <h4 class="episode-grouping-overview-heading">Episode @episodeNumber</h4>
+                    <span>@treatmentEvents.First().FormattedEventDate
+                        @if (episodeNumber != Model.GroupedTreatmentEvents.Count)
```

Comment:
This logic doesn't seem to take into account if the last event in the last episode is episode ending - e.g. `Died`. If that's the case then it should probably give the end date.